src/planwise/component/maps.clj

```diff
@@ -21,7 +29,7 @@
   (digest/sha-256
     (str/join "_" (cons region-id polygons-with-capacities))))
 
-(defn- default-capacity
+(defn default-capacity
```

Comment:
`default-capacity` is defined twice with different bodies. Which one is the correct?